plugins/view/HistogramView/HistogramView.cpp

```diff
@@ -618,7 +618,12 @@ void HistogramView::refresh() {
 }
 
 void HistogramView::graphChanged(Graph *) {
-  setState(DataSet());
+  DataSet old_ds = state();
```

Comment:
Just a nitpick, you should use camel case for the variable names for more consistency with the code style